### DISCUSSION TITLE
General notes Meeting #1 (Feb 28th)

### DISCUSSION THREAD

**Discussion by wolframkriesing** [jscraftcamp/website#1022] [General]
# Location/Date
- codecentric offers their location
- codecentric can handle up to 100 people
- they can sponsor a security person to check 3G certificate
- no rent

# Food, Drinks, etc.
- Joern will check with a caterer
- codecentric has 2 coffee machines
- maybe we do fully vegan?
- cc can offer a BBQ

# Sessions
- session can also be offered to be online
- zoom is available
- session owner can determine: 1) can be online too 2) can be recorded

**Comment by coderbyheart** [comment]
GitHub has a [Discussions](https://docs.github.com/en/discussions) feature. Consider moving content like this (which is not an actionable issue), to discussions.

**Comment by wolframkriesing** [comment]
oh nice, didnt see that yet. thanks

**Comment by coderbyheart** [comment]
If enabled for this project, there is also a link to convert it to a discussion:

![image](https://user-images.githubusercontent.com/188915/156151661-74014988-402e-403b-b705-c3a840b86197.png)

**Comment by coderbyheart** [comment]
Did you agree on a date, yet?

**Reply by wolframkriesing** [reply]
nope, we are voting on it in slack channel #jscc22-orga feel free to join

**Reply by wolframkriesing** [reply]
if you want to join ... https://join.slack.com/t/jscraftcamp/shared_invite/zt-bkvd2305-r1yuHijaYuQnSOeeGFliiQ

**Comment by lowsky** [comment]
This discussion is outdated, so it might be more confusing than helping. 
So I think closing could be the best option at the moment.
Feel fee to re-open or start a new discussion if needed.
